app/src/main/java/ar/com/wolox/android/cookbook/tests/TestLoginRecipeFragment.kt

```diff
@@ -0,0 +1,59 @@
+package ar.com.wolox.android.cookbook.tests
+
+import android.content.Intent
+import ar.com.wolox.android.cookbook.R
+import ar.com.wolox.android.cookbook.datasync.DataSyncRecipeActivity
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import ar.com.wolox.wolmo.core.util.ToastFactory
+import kotlinx.android.synthetic.main.fragment_test_login.vTestLoginEmailInput
+import kotlinx.android.synthetic.main.fragment_test_login.vTestLoginLoginBtn
+import kotlinx.android.synthetic.main.fragment_test_login.vTestLoginPasswordInput
+import javax.inject.Inject
+
+class TestLoginRecipeFragment : WolmoFragment<TestLoginRecipePresenter>(), TestLoginRecipeView {
```

Comment:
This `Fragment` is not very `MVP`-esque. All the validations and business logic decisions are on the `Fragment` side.